test/Microsoft.Extensions.Configuration.Binder.Test/Microsoft.Extensions.Configuration.Binder.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;net46</TargetFrameworks>
```

Comment:
If there's exactly one, we can use `TargetFramework`. More than one as in this case requires the plural `TargetFrameworks`. That said, @davidfowl  and I are debating if we need this change. Most of the server pieces have desktop specific code that we won't be able to stop cross-compiling for until we get to `netstandard2.0`. Mvc also is forced to cross compile due to a library dependency requiring `netstandard1.5`. In all with the exception of a few basic libraries, we are still cross compiling for most parts. Just threw the PR out for @davidfowl to look at.